{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -31,7 +31,19 @@ clean:
 	find . -type f -name "*.py[co]" -delete
 	find . -type d -name "__pycache__" -delete
 
-## Lint using flake8 and black (use `make format` to do formatting)
+{% if cookiecutter.linting_and_formatting == 'ruff' %}
+## Lint using ruff (use `make format` to do formatting)
+.PHONY: lint
+lint:
+	ruff check {{ cookiecutter.module_name }}
+
+## Format source code with ruff
+.PHONY: format
+format:
+	ruff check --select I --fix {{ cookiecutter.module_name }}
```

Comment:
I don't like `--select I` here. We should use the `fixable` or `unfixable` configuration in `pyproject.toml`.

Reply:
Agreed on removing --select. Do you have an opinion on whether we should keep the default (everything is fixable) or specify fixable vs unfixable categories?

Reply:
In my other projects so far, I've been setting `F` as unfixable and it's been working for me so far. I haven't looked that hard at the rulesets though. 

https://github.com/drivendataorg/erdantic/blob/664a3f71e22225635bc95e26578312c9b19dd363/pyproject.toml#L54-L60